services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -58,7 +63,10 @@ const Asset = ({ libraryData }) => {
 
   const [assetData] = libraryData.assets
   const { name, description, inherits: inheritsData, thumbnailData: imageData } = assetData.content
+  const assetFramework = assetData.content.framework
+  const framework = assetFramework.charAt(0).toUpperCase() + assetFramework.slice(1)
```

Comment:
Use the `/data/framework.js` object to get display name.

```js
import { framework as frameworkMap } from '@/data/framework'
```

Then

```js
frameworkMap[framework].name
```

to get the display name and not capitalize the first character of the key.